zebrad/tests/acceptance.rs

```diff
@@ -351,3 +351,61 @@ fn valid_generated_config(command: &str, expected_output: &str) -> Result<()> {
 
     Ok(())
 }
+
+#[test]
+fn listener_address_test() -> Result<()> {
+    listener_address(
+        "localhost:8233",
+        Some("Opened Zcash protocol endpoint at 127.0.0.1:8233"),
+    )?;
+    listener_address(
+        "127.0.0.1:8233",
+        Some("Opened Zcash protocol endpoint at 127.0.0.1:8233"),
+    )?;
+    listener_address(
+        "::1:8233",
+        Some("Opened Zcash protocol endpoint at \\[::1\\]:8233"),
```

Comment:
Some test machines might not have IPv6, so we should also accept failure to bind here.
```suggestion
        Some("Opened Zcash protocol endpoint at \\[::1\\]:8233"),
```